apache-tika-3.0.yaml

```diff
@@ -63,9 +64,9 @@ test:
   pipeline:
     - name: "start server and verify endpoint"
       runs: |
-        TIKA_JAR=$(ls /tika-server-standard-*.jar 2>/dev/null | head -n 1)
+        TIKA_JAR=$(ls /tika-server-standard.jar 2>/dev/null | head -n 1)
```

Comment:
I think we can hardcode this now to `tika-server-standard.jar` after the symlink change was introduced. 💭 

`ls /tika-server-standard.jar 2>/dev/null | head -n 1` this will always return the same name.

Reply:
definitely!